### PR TITLE
Update initialDelaySeconds in kube-apiserver.yaml

### PR DESCRIPTION
**Pull request by utsavanand2**
Signed-off-by: Utsav Anand <utsavanand2@gmail.com>

### CODE REVIEW

**Comment by utsavanand2** [comment]
This may reduce the failures in some installations on the Raspberry PI

File: GUIDE.md [hunk 1]
```diff
@@ -147,7 +147,8 @@ Note: This step can take a long time, even up to 15 minutes.
 Sometimes this stage can fail, if it does then you should patch the API Server to allow for a higher failure threshold during initialization around the time you see `[controlplane] wrote Static Pod manifest for component kube-apiserver to "/etc/kubernetes/manifests/kube-apiserver.yaml"`
 
 ```
-sudo sed -i 's/failureThreshold: 8/failureThreshold: 20/g' /etc/kubernetes/manifests/kube-apiserver.yaml
+sudo sed -i 's/failureThreshold: 8/failureThreshold: 20/g' /etc/kubernetes/manifests/kube-apiserver.yaml && \
+sed -i 's/initialDelaySeconds: [0-9]\+/initialDelaySeconds: 360/' /etc/kubernetes/manifests/kube-apiserver.yaml
```
[the remark this answers is not in the document]
Comment (utsavanand2) [reply GUIDE.md]: I think it should work now.

**Comment by utsavanand2** [comment]
How can I test these changes? Do I need to start everything from scratch or is there any other way of doing things that I'm not aware of?

**Comment by alexellis** [comment]
Sounds like a reasonable approach to me 👍

**Comment by alexellis** [comment]
I tested this out - flannel is failing now but Weave net worked.

**Comment by utsavanand2** [comment]
When I tested this out a while back on my RPi master node, on Hypriot OS, Flannel worked and Weave Net started failing. I'll try the installation again on of my spare Pi's (both on Hypriot and Raspbian) and will let you know :)

**Pull request merged**